File: Web/KombiCim.Data/Models/Arduino/Dtos/LocationDto.cs
using Newtonsoft.Json;

namespace Kombicim.Data.Models.Arduino.Dtos
{
    public class LocationDto
    {
        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public int? Id { get; set; }
        public string DeviceId { get; set; }
        public string DeviceTypeName { get; set; }
        public string Name { get; set; }


        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public List<MinTemperatureDto> MinTemperatures { get; set; }

        [JsonProperty(NullValueHandling = NullValueHandling.Ignore)]
        public double? MinTempValue { get; set; }
    }
}

File: Android/app/src/main/java/com/kc/kombicim/Model/Dto/LocationDto.kt
package com.kc.kombicim.Model.Dto

import com.google.gson.annotations.SerializedName

data class LocationDto (

    @SerializedName("Id") val id : Int,
    @SerializedName("Name") val name: String,
    @SerializedName("DeviceId") val deviceId : String,
    @SerializedName("DeviceTypeName") val deviceTypeName : String,
    @SerializedName("MinTempValue") val minTempValue : Double?
)